spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/serdes.adoc

```diff
@@ -400,9 +400,10 @@ ConsumerFactory cf = new DefaultKafkaConsumerFactory(myConsumerConfigs,
     new RetryingDeserializer(myUnreliableValueDeserializer, retryTemplate));
 ----
 
-Starting with version `3.1.2`, a `RecoveryCallback` can be set on the `RetryingDeserializer` optionally.
+A recovery callback be set on the `RetryingDeserializer`, to return a fallback object
+if all retries are exhausted.
```

Comment:
We prefer follow `One sentence per line` rule: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
Sorry that's an oversight on my end.